bounties_api/notifications/email.py

```diff
@@ -169,7 +187,9 @@ def __init__(self, **kwargs):
             'rating_color': review and Email.rating_color(review.rating),
             'comment': comment and comment.text,
             'MC_PREVIEW_TEXT': preview_text,
-            'rating_url': rating_url
+            'rating_url': rating_url,
+            'average_rating': usd_decimals(average_rating),
```

Comment:
Two decimals and removing any zeroes makes sense here, though it looks weird. Maybe the naming should reflect the decimals instead of one intended use for the rounding.